### ISSUE TITLE
Can you file your example as a separate issue, @jeksterslab?

### ISSUE THREAD

**Issue by jeksterslab** [r-lib/lintr#2035]
Can you file your example as a separate issue, @jeksterslab?
Your case seems to be a more clear-cut false positive.

lintr 3.1.0 expects indentation to be

```r
foo <- function(pkg = c(
  "styler", "lintr", "usethis",
  "tidyverse", "ggplot2", "rmarkdown", "knitr"
)) {
  print(pkg)
}
```

which looks wonky to me.

_Originally posted by @AshesITR in https://github.com/r-lib/lintr/issues/2011#issuecomment-1661957104_
